public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -119,6 +120,16 @@ class UnConnectedExploreToolbar extends PureComponent<Props> {
       contextSrv.hasAccess(AccessControlAction.DashboardsCreate, contextSrv.isEditor) ||
       contextSrv.hasAccess(AccessControlAction.DashboardsWrite, contextSrv.isEditor);
 
+    const isLargerExploreId = largerExploreId === exploreId;
+
+    const onClickResize = () => {
+      if (isLargerExploreId) {
+        this.props.evenResizePane(true);
```

Comment:
AFAICT this is the only place where this is used and the parameter doesn't seem to do anything, is it a leftover?

Reply:
Nope. If you hit the resize button (either + or -) and the panel is larger, it will resize the panes to be even. The prop is calling a state function to do that.

Reply:
but it's called only here and only with `true`, so is it passing `true` really necessary?

Reply:
Good catch! Fixed